Src/BootCamp.Chapter/LetterConverter.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class LetterConverter
+    {
+        public static void Demo()
+        {
+            char keyStroke = RequestKeyStroke();
+            while (IsValidKey(keyStroke))
+            {
+                char arrow = ConvertKeyToArrow(keyStroke);
+                DisplayResult(arrow);
+                keyStroke = RequestKeyStroke();
+            }
+            Console.WriteLine("You pressed an incorrect key. Keep practicing!");
+
+        }
+
+        private static char RequestKeyStroke()
+        {
+            Console.WriteLine("Please enter W, A, S or D to practice your First Person Shooter moves.");
+            return Console.ReadKey(true).KeyChar;
+        }
+
+        private static bool IsValidKey (char keyStroke)
+        {
+            char[] expectedKeys = { 'w', 'W', 'A', 'a', 's', 'S', 'd', 'D' }; 
+            foreach (char letter in expectedKeys)
+            {
+                if (keyStroke == letter)
+                    return true;
+            }
+            return false;
+
+        }
+        private static char ConvertKeyToArrow (char keyStroke)
+        {
+            if (char.ToUpper(keyStroke) == 'W')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5011723)